server/controllers/reports_proposed/templates/style/reports.css

```diff
@@ -39,11 +39,15 @@ tr.outline {
   border-bottom : 1px solid #9f9f9f;
 }
 
-.center > * {
+.header_left {  
+  float: left;
+  width: 40%
+}
+
+.header_center {  
+  float: left;
+  width: 40%;
```

Comment:
header_center === header_left, except for `text-align:center`.  It would probably be better to create an addon class (`.center-text`, for example) that you can just combine in the HTML, rather than create a whole new class.